src/streaming.ts

```diff
@@ -1,13 +1,14 @@
 import EventEmitter from 'events';
-import { AxiosPromise } from 'axios';
-import { ClientApi, FeatureConfig, Segment } from './openapi';
-import { StreamEvent, Options, StreamMsg } from './types';
-import { Repository } from './repository';
-import { ConsoleLog } from './log';
+import {AxiosPromise} from 'axios';
+import {ClientApi, FeatureConfig, Segment} from './openapi';
+import {StreamEvent, Options, StreamMsg} from './types';
+import {Repository} from './repository';
+import {ConsoleLog} from './log';
```

Comment:
Did you turn off Prettier?

Reply:
Ooops. I am not sure - I might not have it installed and set as the formatter of choice. I will fix now.

Reply:
That is really ugly formatting. Headdesk.

Reply:
I copy-paste a config between projects and Prettier is the first thing I ever add. I hate having to think about formatting.

Reply:
Donezel washington now